helm/cas-registration/templates/reporting/rep-deployment.yaml

```diff
@@ -0,0 +1,38 @@
+{{- if (.Values.reportingFrontend.enabled) }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "cas-registration.fullname" . }}-reporting-frontend
+  labels:
+    component: reporting-frontend
+{{- include "cas-registration.labels" . | nindent 4 }}
+spec:
+{{- if not .Values.reportingFrontend.autoscaling.enabled }}
+  replicas: {{ .Values.reportingFrontend.replicaCount }}
+  {{- end }}
+  selector:
+    matchLabels:
+      {{- include "cas-registration.selectorLabels" . | nindent 6 }}
+      component: reporting-frontend
+  strategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+{{- include "cas-registration.selectorLabels" . | nindent 8 }}
+        component: reporting-frontend
+    spec:
+      containers:
+        - name: {{ template "cas-registration.fullname" . }}-reporting-frontend
+          env:
+            - name: API_URL
+              value: http://{{ include "cas-registration.fullname" . }}-backend.{{ .Release.Namespace }}.svc.cluster.local:{{ .Values.backend.service.port }}/api/
+          image: "{{ .Values.reportingFrontend.image.repository }}:{{ .Values.defaultImageTag | default .Values.reportingFrontend.image.tag }}"
```

Comment:
It's done that way in the templates because we replace `defaultImageTag` with the current commit hash in the `make` target. Registration frontend and backend both use the same pattern.